launch_app.sh

```diff
@@ -1,45 +1,38 @@
 #! /usr/bin/env bash
 
-# NAME: launch_app.sh
+# SCRIPT NAME: launch_app.sh
 #
-# BRIEF: This script installs packages needed in the environment to run the
-# See Stuff application and then runs it locally.
+# DESCRIPTION: This script installs packages needed in the environment to run
+# the See Stuff app and then runs it locally. The Dockerfile calls this script.
 #
 # AUTHOR: Kim Lew
 
 set -e
 
+# APP_DIR="/home/ubuntu/rails_see_stuff" # This is root of AWS EC2 instance.
+APP_DIR="/opt/rails_see_stuff" # This is root of Docker container.
+
 check_current_directory () {
   if [[ ! -d '../rails_see_stuff' ]]; then
     echo 'You must be in the directory, rails_see_stuff, after cloning the code'
```

Comment:
Do you think you really need this function anymore? Because of the `set -e` earlier in the script, if the "$APP_DIR" didn't exist, then the command

```
cd "${APP_DIR}"
```

would exit with a non-zero status code, and so your script would exit at that point without ever calling `check_current_directory`.

Reply:
Confirmed you deleted it.